addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-service.js

```diff
@@ -390,6 +395,14 @@ class EnvironmentScService extends Service {
     // Validate input
     await validationService.ensureValid(environment, createSchema);
 
+    // If the AppStream feature is enabled, verify that update request doesn't include a cidr
```

Comment:
Why do we need this check in environment create? I would've thought it only needs to be in environmentScCidrService.update()

Reply:
While environment provisioning we do have a cidr field, so I added the check